app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
We will need this but are you using it at the moment? If not, I would remove it.